packaging/installer/kickstart.sh

```diff
@@ -18,7 +18,7 @@
 #  --reinstall                if an existing install would be updated, reinstall instead
 #  --disable-telemetry        opt out of anonymous statistics
 #  --claim-token              specify a token to use for claiming the newly installed instance
-#  --claim-url                specify a URL to use for claiming the newly installed isntance
```

Comment:
Let's exclude this change. Every time we do changes to `kickstart.sh` we need to update the script checksum in our [docs](https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart#verify-script-integrity), upload the new script to http://my-netdata.io/, etc.